camera_matrix:
  - [700.0, 0.0, 640.0]
  - [0.0, 700.0, 360.0]
  - [0.0, 0.0, 1.0]

dist_coefficients: [0.1, -0.25, 0.0, 0.0, 0.0]
